docker/ubuntu_16.04-i686.dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:16.04
```

Comment:
Just out of curiosity: There are [newer Ubuntu image versions available](https://hub.docker.com/_/ubuntu) (which are even smaller in size), why did you settle on the oldest one?

Reply:
A kind of mental inertia, I guess: Travis CI uses 16.04 now, so that's what I picked. Since the new LTS, 18.04, is out, we could use that instead. I'll go try that.

Reply:
Yep, 18.04 works fine locally, so I pushed the change here as well. Even though the base image is twice smaller than 16.04, the result is still 1.65 Gb. :(